WordPress/gutenberg
# Remove absolute position from svgs

When an SVG is set as a post's featured image, it overflows the image container. This is because the browser doesn't recognize an svg's height. Since theimage is positioned absolutely, the broswer allocates 0 height to the svg, and the image container collapses. This only happens with svgs, so the absolute position attribute is removed from that image type.

## Description

## How Has This Been Tested?
- Create a test post using Gutenberg.
- Upload a jpg image as featured image. 
- Observe that it looks just fine on the page. 
- Allow SVG media uploads by adding the following snippet to your functions.php:

```function mime_types($mimes) {
    $mimes['svg'] = 'image/svg+xml';
    return $mimes;
}
add_filter('upload_mimes', 'mime_types');
```
- Upload an svg image.
- Set it as featured image on your test post. 
- Observe that it overflows the featured image container and looks terrible.
- Apply my patch; run npm; refresh.
- Observe that the svg does not overflow the container anymore.
- Replace it with a jpg again.
- See that the jpg looks okay as well.

## Screenshots (jpeg or gifs if applicable):
Before:
![screen shot 2018-03-12 at 15 21 24](https://user-images.githubusercontent.com/310102/37312729-6c1cb8be-2609-11e8-9d72-d22403cbce6d.png)

After:
![screen shot 2018-03-12 at 15 23 25](https://user-images.githubusercontent.com/310102/37312737-72f7d588-2609-11e8-99e9-a3b110b4be6c.png)

## Types of changes
Bug fix - solves #5477 

## Checklist:
